src/validation/support/contactUs.ts

```diff
@@ -1,18 +1,26 @@
+import { Translation } from '@constants/translations'
+import i18next from 'i18next'
 import { object, string } from 'yup'
 
 export const contactUsValidationSchema = object({
   fullName: string()
-    .matches(/^[aA-zZ\s]+$/, 'Only alphabets are allowed for this field')
-    .required('Full name is required'),
+    .matches(
+      /^[aA-zZ\s]+$/,
+      i18next.t(Translation.PAGE_CONTACT_US_VALIDATION_NAME_MATCHES) as string
+    )
+    .required(i18next.t(Translation.PAGE_CONTACT_US_VALIDATION_NAME) as string),
   email: string()
     .email()
-    .matches(/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/, 'Invalid Email address')
-    .required('Email is required'),
+    .matches(
+      /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/,
```

Comment:
Let's move to constants, not sure why we are validating mail with regex

Reply:
done